src/compressions.jl

```diff
@@ -64,14 +64,15 @@ function _right_sweep_SVD!(ψ::AbstractMPS, Dcut::Int=typemax(Int))
         @cast A[x, σ, y] |= U[(x, σ), y] (σ:d)
         ψ[i] = A
     end
+    ψ[end] *= V[]
```

Comment:
This syntax does not work for CuArrays, change it to trace instead.

Reply:
Done.